apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V102__rda_0_12_0_updates.sql

```diff
@@ -0,0 +1,34 @@
+-- As part of the 0.12 RDA API update, new claim level fields added and revenue lines support added
+
+alter table rda.fiss_claims add received_date_text varchar(10);
+alter table rda.fiss_claims add curr_tran_date_text varchar(10);
+alter table rda.fiss_claims add drg_cd varchar(4);
+alter table rda.fiss_claims add group_code varchar(2);
+alter table rda.fiss_claims add clm_typ_ind varchar(1);
+
+/*
+ * fiss_revenue_lines
+ */
+CREATE TABLE rda.fiss_revenue_lines (
+    dcn varchar(23) NOT NULL,
+    rda_position smallint NOT NULL,
+    non_bill_rev_code varchar(1) NOT NULL,
+    rev_cd varchar(4),
+    rev_units_billed integer,
+    rev_serv_unit_cnt integer,
+    serv_dt_cymd date,
+    serv_dt_cymd_text varchar(10),
+    hcpc_cd varchar(5),
+    hcpc_ind varchar(1),
```

Comment:
Curious on the use of varchar(1) (or (2) for that matter); generally the 'breakeven' point for varchar is (3).  Char(1) or char(2) is more efficient for processing AND storage.

Reply:
According to the [postgresql documentation](https://www.postgresql.org/docs/current/datatype-character.html) there is no performance difference between char and varchar.  Also varchar allows empty strings, which char does not.

Of course this is assuming that AWS's postgresql uses the same storage layout for character data as postgresql.

Reply:
brian, as always, interacting with you is a great learning experience; need to throw out some of my db knowlege from non-postgres db(s)! Ran the following example to prove to myself

```
drop table foo;
CREATE TEMP TABLE foo (id int, v_char char(1) not null, v_small varchar(1), v_big varchar(255));
insert into foo values(1, 'a', 'b', 'testing...123');

SELECT pg_column_size(id)        AS id
     , pg_column_size(v_char)    AS v_char
	 , pg_column_size(v_small)   AS v_small
     , pg_column_size(v_big)     AS v_big
     , pg_column_size(foo)       AS foo
FROM   foo
union all
SELECT pg_column_size(1)
     , pg_column_size('a'::char)
     , pg_column_size('b'::varchar)
     , pg_column_size('testing...123'::varchar)
     , pg_column_size(ROW(1, 'foo'::varchar, '12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890'::varchar));
```

Reply:
Nice experiment!  I just learned this recently myself so don't give me too much credit. :-)